src/Renderer/UI/UpdateHelpers.fs

```diff
@@ -444,13 +475,26 @@ let processContextMenuClick
         model
         |> set (sheet_ >-> SheetT.selectedComponents_) [sym.Id]
         |> withNoCmd
+    
+    | DBScalingBox selectedcomps, "Flip Vertical (Ctrl-Up)"->
+        flipDispatch SymbolT.FlipVertical
+        model 
+        // |> set (sheet_ >-> SheetT.Action_) SheetT.Idle
+        |> withMsg (Msg.Sheet (SheetT.Msg.Wire (BusWireT.Msg.UpdateConnectedWires selectedcomps)))
```

Comment:
This is a matter of judgement but since a lot of BusWire messages are used here, it is a good idea to open BuswireT (and maybe also SymbolT). Then messages can be used without prefix. Although i think it is could practice to indicate module for (unusual) out-of-module functions 1. the draw blokc is sort of all one module and 2. messages are a bit different - well understood. Then, without the prefixes, much of this code will be more readable.

Reply:
One more comment
```
       let rotateCWSym = makeButton RotateCWButton (topleft + CWOffSet)
                                        |> makeRotateSym
                let rotateACWSym = {makeButton RotateACWButton (topleft + ACWOffSet) with SymbolT.STransform = {Rotation=Degree90 ; flipped=false}}
                                        |> makeRotateSym
```

The formatting here is horrible. Do a line break after the `let` = and line up the pipeline. it will be much easier to read and help prevent lines being too long.